src/components/Modal.js

```diff
@@ -53,7 +53,7 @@ const Modal = ({ onDismiss, title, titleExtras, children, width = 450, showCance
       nodeToFocus.focus()
     },
     style: { overlay: styles.overlay, content: { ...styles.modal, width } },
-    aria: { labelledby: titleId },
+    aria: { labelledby: titleId, modal: true },
```

Comment:
This is a great change, but also a separate PR unless essential to this functionality.

Reply:
Sure. I'll pull this and the other changes out into a separate PR handling the first pass at accessibility improvements. These particular roles and keyboard actions impact the accessiblity of the new uploads page, which is why I included them, but there are no particular dependencies that these be in there for the first pass.